costmap_2d/CMakeLists.txt

```diff
@@ -60,8 +61,6 @@ generate_dynamic_reconfigure_options(
 catkin_package(
     INCLUDE_DIRS
         include
-        ${EIGEN_INCLUDE_DIRS}
-        ${PCL_INCLUDE_DIRS}
```

Comment:
Because there already is a catkin_package(DEPENDS PCL Eigen). This means that PCL + Eigen are already added as dependencies for any project that depends on `costmap_2d`, and it's Eigen's (resp. PCL's) job to export their own include dirs. The `catkin_package(INCLUDE_DIRS)` should only export our own include dirs.

Removing these two lines makes the following `catkin_lint` error go away:
```
costmap_2d: CMakeLists.txt(61): error: catkin_package() exports non-package include path
     * You listed one or more include paths in the INCLUDE_DIRS stanza
     * of your catkin_package() call which are not part of your
     * package. If you want to export include paths of other modules,
     * use find_package(), find_path(), and/or find_library() and add
     * the dependency to the DEPENDS stanza.
```